master.yaml

```diff
@@ -0,0 +1,34 @@
+---
+# mayaonline can parse this yaml to obtain tests specific to applications (based on tag) & populate test params,
+# fulfill pre-requisites etc, before deployment. The params are explicitly typed to prompt right values from user.
+# Test parameters can be of different kinds such as ENV, configmap etc., with their own properties.
+# '@userdata' is a placeholder which is either autofilled/userfilled. If empty, use defaults. The test has tags which
+# will help with categorization & application-mapping
+
+
+tests:
+  - name: pod_kill
+    test_tags: ['mysql', 'pumba']
+    location: ./apps/percona/tests/mysql_data_persistence/run_litmus_test.yaml
+    test_params:
+      - name: DB_SERVER_IP
+        kind: env
+        type: string
+        value: 10.20.10.100
```

Comment:
I didn't named this `defaultValue` because as per our flow at some point of time it will contain actual value (provided by user) as well.